From b5f05fe6c83937b8a52c8bf530580d774fcf4ccc Mon Sep 17 00:00:00 2001
From: Christian Hewitt <christianshewitt@gmail.com>
Date: Tue, 3 Aug 2021 18:36:32 +0000
Subject: [PATCH 04/14] FROMLIST(v1): board: amlogic: odroid-n2: fix fdtfile
 suffix for n2-plus

The N2+ dtb is meson-g12b-odroid-n2-plus.dtb, not n2_plus, so
correct the suffix provided in the board file. Also align the
board info string shown during boot to match.

Fixes: 8bc780106c13 ("board: amlogic: odroid: add runtime detection of the N2/N2+/C4/HC4 variants")
Signed-off-by: Christian Hewitt <christianshewitt@gmail.com>
---
 board/amlogic/odroid-n2/odroid-n2.c | 2 +-
 configs/odroid-n2_defconfig         | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/board/amlogic/odroid-n2/odroid-n2.c b/board/amlogic/odroid-n2/odroid-n2.c
index a571eaf162..c94050c776 100644
--- a/board/amlogic/odroid-n2/odroid-n2.c
+++ b/board/amlogic/odroid-n2/odroid-n2.c
@@ -48,7 +48,7 @@ static struct meson_odroid_boards {
 	/* OdroidN2 rev 2019,2,7 */
 	{ MESON_SOC_ID_G12B, 330 * 4, 350 * 4, "n2" },
 	/* OdroidN2plus rev 2019,11,20 */
-	{ MESON_SOC_ID_G12B, 410 * 4, 430 * 4, "n2_plus" },
+	{ MESON_SOC_ID_G12B, 410 * 4, 430 * 4, "n2-plus" },
 	/* OdroidC4 rev 2020,01,29 */
 	{ MESON_SOC_ID_SM1,   80 * 4, 100 * 4, "c4" },
 	/* OdroidHC4 rev 2019,12,10 */
diff --git a/configs/odroid-n2_defconfig b/configs/odroid-n2_defconfig
index 065ecbcfb0..d3f841179e 100644
--- a/configs/odroid-n2_defconfig
+++ b/configs/odroid-n2_defconfig
@@ -9,7 +9,7 @@ CONFIG_DEFAULT_DEVICE_TREE="meson-g12b-odroid-n2"
 CONFIG_MESON_G12A=y
 CONFIG_DEBUG_UART_BASE=0xff803000
 CONFIG_DEBUG_UART_CLOCK=24000000
-CONFIG_IDENT_STRING=" odroid-n2/n2_plus"
+CONFIG_IDENT_STRING=" odroid-n2/n2-plus"
 CONFIG_DEBUG_UART=y
 CONFIG_OF_BOARD_SETUP=y
 # CONFIG_DISPLAY_CPUINFO is not set
-- 
2.17.1

